Library/Homebrew/cmd/deps.rb

```diff
@@ -43,6 +43,11 @@ def deps_args
       switch "--tree",
              description: "Show dependencies as a tree. When given multiple formula arguments, "\
                           "show individual trees for each formula."
+      switch "--digraph",
+             description: "Show dependencies as a directed graph."
+      switch "--graphviz",
+             depends_on:  "--digraph",
+             description: "Show Graphviz digraph code."
```

Comment:
```suggestion
      switch "--dot",
             depends_on:  "--digraph",
             description: "Show text-based graph description in DOT format."
```
The DOT format, while typically processed with Graphviz, is technically not exclusive to it, I think. See https://en.wikipedia.org/wiki/DOT_(graph_description_language).